### ISSUE TITLE
HexDocs example of "drop superfluous _..." is incorrect

### ISSUE THREAD

**Issue by ktekinay** [adobe/elixir-styler#191]
Apologies if this is the wrong place to mention this.

On HexDocs [here](https://hexdocs.pm/styler/1.0.0/styles.html#drops-superfluous-_-in-pattern-matching), in the second example (styled case statement), the "styled" is identical to "before".

```
# Before
case foo do
  _ = bar -> :ok
end
# Styled
case foo do
  _ = bar -> :ok
end
```

should be:

```
# Before
case foo do
  _ = bar -> :ok
end
# Styled
case foo do
  bar -> :ok <-- this
end
```

**Comment by novaugust** [comment]
thanks! poor copy-pasta. happy to take a PR =)

**Comment by novaugust** [comment]
closed by #192 <3
